apps/events/static/cosmos_events/css/carousel.css

```diff
@@ -0,0 +1,9 @@
+.carousel-image {
+    width: 90vh;
```

Comment:
width should be using `50vw` cause `vh` is a percentage of height.
I think using the `.img-fluid` class from bootstrap is probably more elegant than setting the width though

Reply:
I tried img-fluid, and it doesnt work, though if you can get it to work, that would be great. I'll look into the vw-50 option.